misc/cf-support

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env bash
+
+export WORKDIR=/var/cfengine
+export BINDIR="$WORKDIR/bin"
+export non_interactive=0
+
+if [ $# -gt 1 ]; then
+  echo "usage: $(basename $0) [--yes]"
+  echo '	--yes - Non-interactive use. Assume no ticket number and assume include masterfiles.'
+  exit 1
+fi
+
+if [ "$1" = "-M" ]; then
+  cat <<EOF
+.\"Copyright 2022 Northern.tech AS
+.\"
+.\"This file is part of CFEngine 3 - written and maintained by Northern.tech AS.
+.\"
+.\"This program is free software; you can redistribute it and/or modify it
+.\"under the terms of the GNU General Public License as published by the
+.\"Free Software Foundation; version 3.
+.\"
+.\"This program is distributed in the hope that it will be useful,
+.\"but WITHOUT ANY WARRANTY; without even the implied warranty of
+.\"MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+.\"GNU General Public License for more details.
+.\"
+.\"You should have received a copy of the GNU General Public License
+.\"along with this program; if not, write to the Free Software
+.\"Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA
+.\"
+.\"To the extent this program is licensed as part of the Enterprise
+.\"versions of CFEngine, the applicable Commercial Open Source License
+.\"(COSL) may apply to this file if you as a licensee so wish it. See
+.\"included file COSL.txt.
+e COSL.txt.
+.TH CF-SUPPORT 8 "2022-08-10" "CFEngine" "System Administrator"
+.SH NAME
+cf-support \- create tarball of information to submit for support tickets.
+.SH SYNOPSIS
+.B cf-support
+.RI [ OPTION ]
+.SH DESCRIPTION
+cf-support gathers various details about the system and creates a tarball in the current directory to submit to support. If the system is an enterprise hub then additional details will be gathered and included. The utility will prompt for an optional support ticket number as well as prompt whether to include masterfiles in the tarball.
+
+.SH OPTIONS
+.IP "--yes"
+Non-interactive use. Assume no ticket number and assume include masterfiles.
+.SH CFENGINE
+CFEngine provides automated configuration management of large-scale computer systems. A system administrator describes the desired state of a system using CFEngine policy code. The program \fBcf-agent\fR reads policy code and attempts to bring the current system state to the desired state described. Policy code is downloaded by \fBcf-agent\fR from a \fBcf-serverd\fR daemon. The daemon \fBcf-execd\fR is responsible for running \fBcf-agent\fR periodically.
+.br
+Documentation for CFEngine is available at http://cfengine.com/documentation/.
```

Comment:
maybe docs.cfengine.com?

Also, what about changing all links to httpS? Most of these links redirect to httpS, anyway.

Reply:
mm, I copy/pasted this from elsewhere so we have more to fix in this case.

I'll log a ticket to fix all instances.

Reply:
https://tracker.mender.io/browse/ENT-9115